modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -970,6 +1014,182 @@ sub pipeline_analyses {
       -rc_name => '5GB',
     },
 
+    {
+      -logic_name => 'create_pcp_db',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreateDatabase',
+      -parameters => {
+        source_db   => $self->o('scallop_initial_db'),
+        target_db   => $self->o('pcp_db'),
+        create_type => 'copy',
+        force_drop  => 1,
+      },
+      -rc_name    => 'default',
+      -flow_into  => {
+        1 => ['dump_fasta'],
+      },
+    },
+
+    {
+      -logic_name => 'dump_fasta',
+      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+      -rc_name    => '8GB',
+      -parameters => {
+        cmd => 'mkdir -p ' . $self->o('pcp_dir') . ';'.
```

Comment:
The mkdir command should not be done in the analysis. It should happen in pipeline_create_commands or in a specific analysis which would create all the directory tree of the pipeline and be the first analysis

Reply:
Moved creation of 'pcp_dir' to pipeline_create_commands.